packages/dialtone-tokens/tokens/$themes.json

```diff
@@ -3725,12 +3789,72 @@
   {
     "id": "ae20a00dfcb0ef5937057144425831ea08531d09",
     "name": "dark",
-    "$figmaStyleReferences": {},
+    "$figmaStyleReferences": {
```

Comment:
What is this? 🤔

Seems like you added just two variables and a lot of style references were added (automatically I think), is this expected?

Reply:
Yeah, "automatically" is correct. In creating the new tokens, the Tokens Studio plugin connects them with the Figma-only styles, but it also did that for others that were **not** previously not connected.

I did confirm in the Figma UI the connections result is good.

The `$figmaStyleReferences` don't impact the subsequent transformation from Style Dictionary and whatever else.